File: lesson2/python3/homework/homework12.py
#!/usr/bin/python3
import math

coordinate_x1 = int(input("Введите координата x1 - "))
coordinate_x2 = int(input("Введите координата x2 - "))
coordinate_y1 = int(input("Введите координата y1 - "))
coordinate_y2 = int(input("Ввудите координата y2 - "))

distance = math.sqrt(
    (coordinate_x2 - coordinate_x1) ** 2 + (coordinate_y2 - coordinate_y1) ** 2
)

print(
    "Растояние коордионаты между двух точек на чесловой оси - ", int(distance)
)

File: lesson2/c_c++/homework/homework12.cpp
#include <iostream>
#include <cmath>

int main()
{
    using namespace std;
    int coordinate_x1, coordinate_x2, coordinate_y1, coordinate_y2, distance;

    cout << "Введите координата x1 - ";
    cin >> coordinate_x1;
    cout << "Введите координата x2 - ";
    cin >> coordinate_x2;
    cout << "Введите координата y1 - ";
    cin >> coordinate_y1;
    cout << "Введите координата y2 - ";
    cin >> coordinate_y2;

    distance = sqrt(pow(coordinate_x2 - coordinate_x1, 2) + pow(coordinate_y2 - coordinate_y1, 2));
    cout << "Расстояние между координата двух точек на  числовой оси - " <<  distance << endl;

    return 0;
}

